### ISSUE TITLE
Build faild: failed to resolve: use of undeclared type `MockedBlockchain`

### ISSUE THREAD

**Issue by willser** [near/near-sdk-rs#743]
Hi guys,
 I try to write a unit test to test my code.But when I follow this code to write, I got this error.
 ------------

1.  1.  1.  1.  Unit test code
                 ```rust
                 #<span class="error">[cfg(all(test, not(target_arch = "wasm32")))]</span>
                 mod tests {
                 use super::*;
                 use near_sdk::test_utils::VMContextBuilder;
                 use near_sdk:: {testing_env, VMContext} 

                ;
                 use std::convert::TryInto;

 fn get_context(is_view: bool) -> VMContext 

 { VMContextBuilder::new() .signer_account_id("bob_near".try_into().unwrap()) .is_view(is_view) .build() } 

 #<span class="error">[test]</span>
 fn my_test() 

 { let context = get_context(false); testing_env!(context); // ... Write test here } 

}
 ```

------------

1.  1.  1.  1.  Error
                 ```log
                 error<span class="error">[E0433]</span>: failed to resolve: use of undeclared type `MockedBlockchain`
                 --> src\lib.rs:65:9 

                <div class="table-wrap"> 
                </div>

                65 | testing_env!(context);

                <div class="table-wrap">
                  | ^^^^^^^^^^^^^^^^^^^^^ not found in this scope | 
                </div>

                 = note: this error originates in the macro `testing_env` (in Nightly builds, run with -Z macro-backtrace for more info)
                 help: consider importing this struct
                 ```
                 ------------
                 Following the prompts, I import `MockedBlockchain` to this scope like this:
                 ```rust 
                 use near_sdk::MockedBlockchain;
                 ```
                 It works.
                 Then I'm wonder is there something wrong with the way I use it or this macro needs to be modified.

Some information about my machine:

*   near-sdk = "3.1.0"
*   `rustc` version is `rustc 1.58.0 (02072b482 2022-01-11)
*   cargo 1.58.0 (7f08ace4f 2021-11-24)

**Comment by austinabell** [comment]
yes, this is no longer an issue with the 4.0 pre-releases or what exists in the code now.

This could be fixed in a release branched off the 3.1 commit, so either that or 4.0 stable being released will close this issue.

**Comment by willser** [comment]
Thank you,it's helpful.Looking forward to the 4.0 stable.
